docs/tutorials/storage.md

```diff
@@ -180,5 +183,70 @@ To use an infrastructure block when building a deployment, the process is simila
 
 The `prefect deployment build` command also supports specifying a built-in infrastructure type prepopulated with defaults by using the `--infra` or `-i` options and passing the name of the infrastructure type: `docker-container`, `kubernetes-job`, or `process`.
 
+## Sharing blocks between different deployments
```

Comment:
Some capitalization inconsistencies in this section -- this header has lowercase `blocks` but two headers down has capitalized `Blocks`. I'd vote lowercase everywhere unless referring explicitly to it as a feature name (as in the next sentence, "one of the major benefits of Prefect Blocks")

Reply:
Bigger conversation about what we want to capitalize. Currently we are not capitalizing any core Prefect components unless discussing the class name, for instance. So we have not been capitalizing blocks, agents, work queues, flows, tasks, subflows, and so on. For consistency now blocks should be lowercase.